controllers/suite_util_test.go

```diff
@@ -219,6 +219,18 @@ func createComponent(componentLookupKey types.NamespacedName) {
 	getComponent(componentLookupKey)
 }
 
+// createComponentCustom creates custom component resource and verifies it was properly created
+func createComponentCustom(sampleComponentData *appstudiov1alpha1.Component) {
+	Expect(k8sClient.Create(ctx, sampleComponentData)).Should(Succeed())
+
+	lookupKey := types.NamespacedName{
+		Name:      sampleComponentData.Name,
+		Namespace: sampleComponentData.Namespace,
+	}
+
+	getComponent(lookupKey)
+}
```

Comment:
Ideally, this `createComponentCustom` and existing `createComponent` are merged eventually.

Reply:
I did make createComponent use createComponentCustom

one needs just key another whole component, so merging would mean cover both cases, and I think it is better to have that function simpler